packages/core/auto-export.config.json

```diff
@@ -2,6 +2,7 @@
     "entries": [
         {
             "inputPath": "packages/**/src/index.ts",
+            "ignorePackages": ["test-utils"],
```

Comment:
This is a copy of https://github.com/bigcommerce/checkout-js/pull/1101. Once https://github.com/bigcommerce/checkout-js/pull/1101 is merged, I'll rebase.
This PR's major changes are in the `core` and `hosted-field-integration` folders.